node/node_handler.go

```diff
@@ -443,14 +464,20 @@ func (node *Node) PostConsensusProcessing(
 		node.BroadcastCXReceipts(newBlock, commitSigAndBitmap)
 	} else {
 		if node.Consensus.Mode() != consensus.Listening {
+			sigs := "none"
+			if sigsInBlock, err := node.signaturesIncludedInBlock(newBlock); err == nil {
```

Comment:
While this PR open and you have this code, can you do me a favor and compare the time of compute needed to call this block.

Reply:
between 12-60 microseconds in local settings with one key per node.

Reply:
5-25 microseconds after removing the SerializeToHexStr.

Reply:
sorry - since so easy now - can also say when under multi bls key, if that causes any extra slowdown

Reply:
actually nvm - at the level of microsecond - doesn't matter , all good